test/unit/BPool.t.sol

```diff
@@ -803,13 +809,50 @@ contract BPool_Unit_Unbind is BasePoolTest {
 }
 
 contract BPool_Unit_Gulp is BasePoolTest {
-  function test_Revert_NotBound() private view {}
+  struct Gulp_FuzzScenario {
+    address token;
+    uint256 balance;
+  }
 
-  function test_Revert_Reentrancy() private view {}
+  modifier happyPath(Gulp_FuzzScenario memory _fuzz) {
+    vm.assume(_fuzz.token != VM_ADDRESS);
+    vm.assume(_fuzz.token != 0x000000000000000000636F6e736F6c652e6c6f67);
```

Comment:
What does `0x000000000000000000636F6e736F6c652e6c6f67` represent?

Reply:
It's the console address, had to hardcode it since there is no accesible constant as `VM_ADDRESS`

Reply:
Check https://github.com/foundry-rs/forge-std/pull/229.